### PR TITLE
Mentioned the "Symfony Upgrade Fixer" in the upgrade article

### LINKED ISSUE #5951: [RFC] Symfony Upgrade Fixer in "Upgrading a Major Version"
[Symfony Upgrade Fixer](https://github.com/umpirsky/Symfony-Upgrade-Fixer) analyzes Symfony project and tries to make it compatible with the new version of Symfony framework. It is based on [PHP CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer).

I would like to get some additional feedback and see if people use it and how we can improve.
It is already mentioned in [The new Awesome of Symfony 3.0 Tutorial](https://knpuniversity.com/screencast/new-in-symfony3/form-updates) on KnpUniversity.

If you take a look at [real life examples](https://github.com/umpirsky/Symfony-Upgrade-Fixer#real-life-examples) you will see that some fixers can be serious time savers. So I guess informing people about such tool can give them some wind in the back with the upgrade process.

If you like it and think it will be useful to have it mentioned in [Upgrading a Major Version](https://github.com/symfony/symfony-docs/blob/3.0/cookbook/upgrade/major_version.rst#3-update-your-code-to-work-with-the-new-version) I will be happy to submit a pull request.

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #5951.

### CODE REVIEW

File: setup/upgrade_major.rst [hunk 1]
```diff
@@ -4,10 +4,10 @@
 Upgrading a Major Version (e.g. 2.7.0 to 3.0.0)
 ===============================================
 
-Every few years, Symfony releases a new major version release (the first number
-changes). These releases are the trickiest to upgrade, as they are allowed to
-contain BC breaks. However, Symfony tries to make this upgrade process as
-smooth as possible.
+Every two years, Symfony releases a new major version release (the first number
+changes). These releases are the trickiest to upgrade, as they are allowed break
```
Comment (snoek09) [inline setup/upgrade_major.rst]: allowed `to` break backwards compatibility

**Comment by snoek09** [comment]
Cool stuff :+1: 

One minor comment.

Status: reviewed.

**Review by wouterj — CHANGES_REQUESTED** [review]
Afaik, it only works for Symfony 2 to 3 atm, I think we have to mention that.

**Comment by javiereguiluz** [comment]
@wouterj thanks for the review. I made the change you asked for.

**Review by wouterj — APPROVED** [review]

**Comment by wouterj** [comment]
👍

**Comment by weaverryan** [comment]
Awesome!

**Pull request merged**